utils/scripts/helpers/repo-helper-el9.sh

```diff
@@ -38,6 +40,16 @@ install_curl() {
     :
 }
 
+# Use local repo server if present
+install_optional_ca() {
```

Comment:
Do we have a common/utility shell script we source this from instead of redefining multiple times?

Reply:
Not at this time.   Any sourced script would have to also be "COPY" into the Dockerfile build environment.